### ISSUE TITLE
clarification on label-prefix topology requirements

### ISSUE THREAD

**Issue by delphisinformatica** [IBM/ibm-block-csi-driver#743]
the documentation mentions that:

_(Optional) To use CSI Topology, at least one node in the cluster must have the label-prefix of topology.block.csi.ibm.com to introduce topology awareness.

Important: This label-prefix must be found on the nodes in the cluster before installing the IBM® block storage CSI driver. If the nodes do not have the proper label-prefix before installation, CSI Topology cannot be used with the CSI driver._

does that mean that there should be any label with that  prefix set on at least one node in the cluster? i.e. something like this:

`oc label node worker1.example.com topology.block.csi.ibm.com/topology=true`

**Comment by lechapitre** [comment]
Yes, but of course the topology label should not be "toplogy=true" but should be a label that matches one of the labels in the filter from the secret (if that secret is to be used on that node).

The filter labels are the ones specified in "supported_topologies" in the secret (per managed system) - see example secret in the documentation ('Creating a Secret with topology awareness")

As you can see in the example - you can define multiple filter labels in the secret (per management id)
And you can define multiple toplogy labels for each node.

All filter labels from the secret should exist in the node labels for the secret to be used.

**Comment by kasserater** [comment]
@delphisinformatica any additional issue here or can we close this item?

**Comment by delphisinformatica** [comment]
issue can be closed, thanks for the clarification
